app/dashboard/src/hooks/backendHooks.ts

```diff
@@ -560,3 +575,146 @@ export function useBackendGetOrganization(backend: Backend | null) {
     uploadOrganizationPictureVariables,
   ])
 }
+
```

Comment:
just FYI: we're going to separate query/mutation options from hooks to make it easier to reuse the logic between react and vue.

Reply:
wait :skull: you can't use passive listeners for mutations

Reply:
some notes: for most of the hooks added in this refactor, there are dependencies on both `useBackend` and `useQueryClient`.

- i don't think `useBackend` will be an issue - it's mainly here for convenience as for those specific hooks it will *always* be the `RemoteBackend` and will *never* be the `LocalBackend` - meaning the dependency can be trivially refactored out if we ever wish to use those methods in Vue. i'm not sure how likely that is, i inlined the `useRemoteBackend` hook because i figured it's unlikely Vue will need them anytime soon (if at all.)
- `useQueryClient` *is* an issue though... i'd have expected *some* framework-agnostic way to get it but i guess it doesn't exist? not sure what a good solution is... perhaps it can be moved to a function parameter too.

Reply:
either way i *guess* this implemention is more conventional than the one it replaces, at least? so i assume (assuming no major problems) it shouldn't be a reason this can't go in, at least.

although - if you have a preferred API and/or suggestions then please feel free to make some suggestions

Reply:
upon further thought it's not like the current implementation seems to be causing any issues, so it's not like this PR needs to go in

Reply:
> wait 💀 you can't use passive listeners for mutations

Well, not, only for queries, I guess, though you can find a running mutation by `mutationKey` and subscribe to it